paper/paper.bib

```diff
@@ -53,7 +53,7 @@ @inproceedings{masad2015
 }
 
 @inproceedings{kazil2020,
-  title={Utilizing python for agent-based modeling: The mesa framework},
+  title={Utilizing {P}ython for agent-based modeling: The {M}esa framework},
```

Comment:
Could you explain what the idea is behind the additional brackets?

Reply:
JOSS uses Sentence case for title, so this overrides the rule and preserves capitalization for proper nouns when it otherwise would be lost

Reply:
Perfect, learned something, thanks!